### ISSUE TITLE
Question about freezing updates for model

### ISSUE THREAD

**Issue by max-assel** [YuvalNirkin/hyperseg#5]
Hello, thank you so much for releasing your code! I am currently training the model on a custom dataset, and I was wondering if it is possible to freeze the gradient updates for a select number of classes? I am currently zeroing out the gradients from the decoder level 4 BatchNorm3 layer because that is the only layer that has the size of the number of classes within my custom dataset, but when I attempt this, it appears that the weights of the classes that I try to freeze are still changing.

**Comment by YuvalNirkin** [comment]
What do you mean by "the weights of the classes"? The final fc layer's weights are dynamic.

**Comment by max-assel** [comment]
I see, I misunderstood the model architecture. Thank you!
